.github/ISSUE_TEMPLATE/2_Badge_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: 💡 Badge Request
+about: Ideas for new badges
+
+---
+
+:clipboard: **Description**
+<!--
+A clear and concise description of the new feature.
+
+- Which service is this badge for e.g: GitHub, Travis CI
+- What sort of information should this badge show e.g: version, downloads
+-->
```

Comment:
I think we can add something like:
- Can you give an example badge using plain text (e.g. "users | 213") or by using static badge ![](https://img.shields.io/badge/users-213-brightgreen.svg) (static badge generator can be found at https://shields.io)